File: Archivos Python/PruebaArchivos.py
from abc import ABCMeta

class DesigualdadesAbs(ABCMeta):

    # Método de clase
    @staticmethod
    def Desigualdad(edad,antiguedad):
        if edad > 60 and antiguedad < 25:
            return 1

        elif edad < 60 and antiguedad > 25:
            return 2
        else:
            edad > 60 and antiguedad > 25
            return 3

class pruebaArchivos:

    def leerArchivo(self, archivo):
        file = open(archivo, 'r')
        lineas = []
        lineas_archivo = []
        for linea in file.readlines():
            lineas.append(linea.replace('\n', '').split(";"))
        file.close()
        for f in range(0, len(lineas)):
            try:
                lineas_archivo.append([float(lineas[f][0]),  float(lineas[f][1])])
            except ValueError:
                lineas_archivo.append([0,0,0.0])
        return lineas_archivo

    def calcularDesigualdad(self, lista):
        resultados = []
        for f in range(0, len(lista)):
            resultados.append(DesigualdadesAbs.Desigualdad(
                lista[f][0],lista[f][1]))
        return resultados

    def guardarResultados(self, entrada, resultados):
        file = open("resultados.txt", 'w')
        for f in range(0, len(entrada)):
            linea = str(entrada[f][0]) + '#' + \
                    str(entrada[f][1]) + '#' + \
                    '#' + str(resultados[f]) +'\n'
            file.write(linea)
        file.close()

if __name__ == "__main__":
    prueba = pruebaArchivos()
    archivo = []
    archivo = prueba.leerArchivo("datos.txt")
    print(archivo)
    resultado = prueba.calcularDesigualdad(archivo)
    print(resultado)
    prueba.guardarResultados(archivo, resultado)

File: PRACTICA ARCHIVOS JAVA/pruebaArchivos.java
import java.io.*;
import java.util.ArrayList;

public class pruebaArchivos {

    private ArrayList<String> contenidoArchivo = new ArrayList<>();
    private ArrayList<Desigualdad> listaEntradas = new ArrayList<>();
    private final String rutaWindows = "C:\\Users\\Joserc\\IdeaProjects\\JoselynneCodigo\\src\\datos.txt";
    private final String rutaLinux = "/C:/Users/Joserc/IdeaProjects/JoselynneCodigo/src/datos.txt";

    public ArrayList<String> getContenidoArchivo() {
        return contenidoArchivo;
    }

    public ArrayList<Desigualdad> getListaEntrada() {
        return listaEntradas;
    }

    public void leerArchivo(){
        LeerArchivo leer = new LeerArchivo();
        this.contenidoArchivo =
                leer.leerArchivo(this.rutaLinux);
        for (int i=0; i < this.contenidoArchivo.size();i++){
            String linea = this.contenidoArchivo.get(i);
            String elementosLinea[] = linea.split("#");
            try {
                int Edad = Integer.parseInt(elementosLinea[0]);
                int Antiguedad = Integer.parseInt(elementosLinea[1]);
                listaEntradas.add(new Desigualdad(Edad,Antiguedad));
            } catch (NumberFormatException ex){
                ex.printStackTrace();
                listaEntradas.add(new Desigualdad(0,0));
            }
        }
    }

    public void escribirArchivo(ArrayList<Desigualdad> lista){
        String archivo = "resultados.csv";
        File f = new File(archivo);
        //Escritura
        try{
            FileWriter w = new FileWriter(f);
            BufferedWriter bw = new BufferedWriter(w);
            PrintWriter wr = new PrintWriter(bw);
            wr.write("");
            wr.append("EDAD,ANTIGUEAD, RESULTADO\n");
            for (int i=0; i < lista.size(); i++){
                wr.append(lista.get(i).getEdad() +
                        "," + lista.get(i).getAntiguedad()+ "," +
                        lista.get(i).getResultado() + "\n");
            }
            wr.close();
            bw.close();
        }catch(IOException e){

        }
    }

    public static void main (String args[]){
        pruebaArchivos prueba = new pruebaArchivos();
        prueba.leerArchivo();
        for (int i=0; i < prueba.getListaEntrada().size(); i++){
            System.out.println(prueba.getListaEntrada().get(i).getEdad() +
                    "\t" + prueba.getListaEntrada().get(i).getAntiguedad() +
                    "\t"+  prueba.getListaEntrada().get(i).getResultado());
        }
        prueba.escribirArchivo(prueba.getListaEntrada());

    }
}
